content/blog/how-to-backup-and-restore-stateful-applications-on-kubernetes-using-kasten-k10-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,660 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pvc
+  namespace: mysql
+  labels:
+    app: mysql
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+```
+
+This PVC file, together with other YAML manifest files in the folder *'base'*, will be used to install the MySQL database instance using [Kustomize](https://kustomize.io/).
+
+```shell
+$ tree mysql-app/base
+mysql-app
+/base
+├── kustomization.yaml
+├── mysql-deployment.yaml
+└── mysql-pvc.yaml
+```
+
+The file *kustomization.yaml* lists all YAML files in its resources section, together with the secret generator for MySQL password:
+
+```shell
+$ cat mysql-app/base/kustomization.yaml 
+secretGenerator:
+- name: mysql-pass
+  namespace: wordpress
+  literals:
+  - password=CfeDemo@123
+resources:
+  - mysql-deployment.yaml
+  - mysql-pvc.yaml
+```
+
+T﻿yping below command to install the MySQL database to the namespace *'mysql'*:
+
+```shell
+$ kubectl apply -k mysql-app/base
+namespace/mysql created
+secret/mysql-pass-m62cbhd9kf created
+service/mysql created
+persistentvolumeclaim/mysql-pvc created
+deployment.apps/mysql created
+```
+
+T﻿yping the following command to check the MySQL database deployment state. The MySQL Pod should be in *Running* status:
+
+```shell
+$ kubectl get all -n mysql
+NAME                         READY   STATUS    RESTARTS   AGE
+pod/mysql-6974b58d48-wb8g5   1/1     Running   0          14s
+
+NAME            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+service/mysql   ClusterIP   None         <none>        3306/TCP   24s
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/mysql   1/1     1            1           23s
+
+NAME                               DESIRED   CURRENT   READY   AGE
+replicaset.apps/mysql-6974b58d48   1         1         1       24s
+```
+
+Y﻿ou can check the *PersistentVolume* (PV) and the PVC get provisioned as part of the MySQL database deployment:
+
+```shell
+$ kubectl get persistentvolumes 
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                                                                                                 STORAGECLASS               REASON   AGE
+
+pvc-3e55e9b3-097f-4ddf-bdcb-60825a7905ec   1Gi        RWO            Delete           Bound    mysql/mysql-pvc                                                                                                                           
+
+$ kubectl get persistentvolumeclaims -n mysql
+NAME        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS               AGE
+mysql-pvc   Bound    pvc-3e55e9b3-097f-4ddf-bdcb-60825a7905ec   1Gi        RWO            gl-sbp-frank-gl1-sstor01   9m50s
+```
+
+**2. Access MySQL database**
+
+In order to access MySQL database service using the mysql CLI, set first the port-forward of *service/mysql*:   
```

Comment:
Change the second phrase in the sentence to "...CLI, first set the port-forward..." (swapping the position of the words "first" and "set"